servicetest/capabilities/test_caps.py

```diff
@@ -230,3 +230,17 @@ def test_defaults_are_applied(self):
             assert my_env_var == "TEST_ENV_VALUE"
         finally:
             sc.terminate()
+
+    def test_set_same_cap_twice(self):
+        """ Test that setting the same capability doesn't result in crashes
+        """
+        sc = Container()
+        try:
+            sc.start(DATA)
+            sc.set_capabilities(["test.cap.valid-dbus"])
+
+            with pytest.raises(DBusException) as err:
+                # Set the same capability again
+                sc.set_capabilities(["test.cap.valid-dbus"])
```

Comment:
Does this really cover the testcase? The error to catch will still trigger a DBusException with the text "No response" or something ...